### ISSUE TITLE
Test for offroad power usage

### ISSUE THREAD

**Issue by adeebshihadeh** [commaai/openpilot#2177]
Run a comma two for a few minutes while offroad and check power consumption

**Comment by LoganxDev** [comment]
Still needed?

**Comment by adeebshihadeh** [comment]
Yes, but this is one we'll want to do internally. Feel free to pickup something else like increasing the mypy coverage.
